README.md

```diff
@@ -169,6 +169,15 @@ VALValet *const mySharedValet = [VALValet sharedGroupValetWithGroupPrefix:@"grou
 
 This instance can be used to store and retrieve data securely across any app written by the same developer that has `group.Druidia` set as a value for the `com.apple.security.application-groups` key in the app’s `Entitlements`. This Valet is accessible when the device is unlocked. Note that `myValet` and `mySharedValet` cannot read or modify one another’s values because the two Valets were created with different initializers. All Valet types can share secrets across applications written by the same developer by using the `sharedGroupValet` initializer. Note that on macOS, the `groupPrefix` [must be the App ID prefix](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_application-groups#discussion).
 
+It is possible to create multiple Valets within the same shared access group be specifying the same group identifier but different unique identifiers, like so:
+
+```
+let valet1 = Valet.sharedGroupValet(with: groupID, identifier: Identifier(nonEmpty: "valet1")!, accessibility: accessibility)
+let valet2 = Valet.sharedGroupValet(with: groupID, identifier: Identifier(nonEmpty: "valet2")!, accessibility: accessibility)
+```
+
+These two Valets cannot access each others data, but they are both accessible from all applications or extensions which have access to the given `groupID`. The `identifier` (but not the `groupIdentifier`) is optional on shared Valets. All Valets created within the same group using the default `identifier` can access the same data.
+
```

Comment:
let's move this blurb up to the `Valet` section and then replace this with:
```suggestion
As with Valets, shared iCloud Valets can be created with an additional identifier, allowing multiple independently sandboxed keychains to exist within the same shared group.

```

Reply:
@efirestone we still want this blurb but we want it further up in the README 😉

Reply:
I'm not sure I'm following what you're suggesting (and now the review recommendation can't be committed). Are you saying put this blurb in the `Valet` section:
```
As with Valets, shared iCloud Valets can be created with an additional identifier, allowing multiple independently sandboxed keychains to exist within the same shared group.
```
or to leave that blurb in the "Sharing Secrets" section, and then put the original text up in the Valet section?

Reply:
Ah! Okay I see where I failed to read here. It's the _additional_ in `additional identifier` that threw me.

The _additional_ is referring to iCloud Valets, whereas Valets have a primary identifier.

I think we should just remove the "As with Valets" preamble and this'll make more sense to me.